### ISSUE TITLE
Does not work on Win7

### ISSUE THREAD

**Issue by Tcll** [krishnaraj/oneclipboard#3]
so much for a 2nd attempt at having a 3rd monitor... heh

Yea apparently I can't use my MBd VGA connector with a PCIe card on a dual-core CPU, but I can with a PCI card on a single core CPU...
So after finding out that newer tech can't do what older tech can do, I just tried using my laptop as my 3rd monitor...

the thing starts up and remains active after putting in my ID and pass, but I get no clipboard action between my desktop and laptop.

it works just fine on my WinXP VM though, though as my previous issue stated, it couldn't handle files, and it can't even handle a screenshot.

**Comment by Tcll** [comment]
scratch that, it just now started working...

that was weird, but it works now :)

**Comment by Tcll** [comment]
sorry if this is notifying you like heck...
one moment it's not working, the next moment, it's working, and the next moment it's not working again...

**Comment by krishnaraj** [comment]
Is there anything in the logs? Could be a connectivity issue.

**Comment by Tcll** [comment]
eh... I... dislike java...
thus I never pay attention to that sort of stuff... heh
where would I find the logs?? :/

EDIT:
btw, I doubt it's a connection issue... but I'm not out-ruling it ;)
my laptop has 4 bars on a w/less G connection and my compy has w/less N.

when I can, I'll give my secondary XP machine a try as well.

**Comment by krishnaraj** [comment]
There should be an onclipboard.log in your users directory.

**Comment by Tcll** [comment]
nope, checked in every plausible directory after doing a search...
the only thing found was oneclipboard.lock which only contains an int... heh
